docs/src/adoc/index.adoc

```diff
@@ -7440,6 +7443,71 @@ Global defined attributes may be configured on the factory bean:
 </bean>
 ----
 
+==== Annotation-based configuration
+For annotation based configurations you can add the following method in a
+`@Configuration` class:
+
+[source,java,indent=0]
+----
+@Configuration
+public class JdbiConfiguration {
+    @Bean
+    public Jdbi jdbi(DataSource ds) {
+        ConnectionFactory cf = new SpringConnectionFactory(dataSource); // <1>
+        final Jdbi jdbi = Jdbi.create(cf);
+        /* additional configuration goes here */
+        return jdbi;
+    }
+}
+----
+<1> This connection factory will enable <<jta, JTA support>>.
+
+[#jta]
+==== Synchronization between Jdbi and JTA
+If you use Spring and databases, you will most likely also use the Java Transaction API (JTA)
+to manage the transactions. The `JdbiUtil` utility can be used to obtain link:{jdbidocs}/core/Handle.html[Handles^]
+synchronized with Spring's JTA, as can be seen in the following fragment:
+
+[source,java,indent=0]
+----
+Handle handle = JdbiUtil.getHandle(jdbi); // <1>
```

Comment:
Handle is a closeable object and the general guidance everywhere else in the doc is to either use callbacks (which does not apply here) or use try-with-resources. Could this be rewritten as

```
try (Handle handle = JdbiUtil.getHandle(jdbi)) { 
    /* use the hande */
} 
```

thus not needing the explicit "closeIfNeeded" call.

Reply:
looking through the code, I think that is correct. I don't know enough about Spring to be able to make meaningful comments on the JdbiUtil class.